app/controllers/v0/sessions_controller.rb

```diff
@@ -136,12 +136,7 @@ def user_logout(saml_response)
 
     def login_stats_success(saml_response, user_session_form = nil)
       tracker = url_service(user_session_form&.saml_uuid).tracker
-      tags = [
-        "loa:#{@current_user.loa[:current]}",
```

Comment:
Could you explain the removal of the idp and loa tags? The reason isn't obvious to me based on the PR description

Reply:
I added the `loa` and `idp` tags a [couple of months ago](https://github.com/department-of-veterans-affairs/vets-api/pull/3762).  They're redundant as both can be inferred from the `context` tag, however at the time I thought it would be easier to query on.

After reading the [StatsD Metrics Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/backend/sending-metrics-using-statsd.md#why-do-we-increment-a-bunch-of-metrics-by-0-in-our-initializers) yesterday it became apparent that too many tags can have performance implications in Prometheus and it does make initialization more complex.  So in light of this, and knowing the values are redundant, I removed them.

Reply:
Ok thanks for the info! Just wanted to make sure we weren't breaking any grafana boards by removing them! ➕ 💯 for removing tags in relation to better performance